src/features/device-state/routes/state-patch-v2.ts

```diff
@@ -208,6 +191,32 @@ export const statePatchV2: RequestHandler = async (req, res) => {
 							},
 							body: deviceBody,
 						});
+					});
+				}
+			}
+			if (
+				Object.keys(metricsBody).length > 0 &&
+				(await shouldUpdateMetrics(uuid))
+			) {
+				updateFns.push(async (resinApiTx) => {
+					const latestDeviceMetricsRecord = await resinApiTx.get({
+						resource: 'device_metrics_record',
+						id: { is_reported_by__device: deviceId },
+					});
+					if (latestDeviceMetricsRecord == null) {
+						await resinApiTx.post({
+							resource: 'device_metrics_record',
+							id: { is_reported_by__device: deviceId },
+							body: {
+								...metricsBody,
+							},
+						});
```

Comment:
Does that work? Shouldn't it be v ?
```suggestion
						await resinApiTx.post({
							resource: 'device_metrics_record',
							body: {
								...metricsBody,
								is_reported_by__device: deviceId,
							},
						});
```